scheduler/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": [
+    {
+      "name": "scheduler.metrics.port",
```

Comment:
should this be `healthserver`?

Reply:
yeah we missed committing that by mistake.
This particular file is more for editors and verifiers on the properties files thats why no tests picked it up.